snapshot-labs/sx-ui
# feat: add strategies editing

Closes: https://github.com/snapshot-labs/sx-ui/issues/659

This PR adds ability to edit authenticators/votingStrategies/proposalValidationStrategy on spaces (EVM and Starknet).

## Test plan
- Deploy new space (to make sure it has all metadata created properly, old spaces didn't have proposal validation strategies metadata).
- Try editing it.

## Screenshots

Temporary UI, but functional.

<img width="1028" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/6b866ba1-6d57-47d0-993f-e9fd842ec4fb">
